README.md

```diff
@@ -277,61 +255,52 @@ pre-requisites...
 
 * Golang 1.15 or newer
 * Docker, either the docker.io packages on Ubuntu, Docker Desktop for Mac,
-  or your preferred docker distribution.
-* [mage](https://magefile.org/): There are some tips for using mage in
-  [docs/developer/mage.md](docs/developer/mage.md)
+  or your preferred docker distribution. Other container engines such as podman should work also.
 
 ### Building
 
 The operator uses [mage](https://magefile.org/) for its build process.
 
 #### Build the Operator Container Image
-This build task will create the operator container image, building or rebuilding
-the binary from golang sources if necessary:
+This build task will create the operator container image, building or rebuilding the binary from golang sources if necessary:
 
 ``` bash
-mage operator:buildDocker
+make
 ```
 
 #### Build the Operator Binary
 If you wish to perform ONLY to the golang build or rebuild, without creating
 a container image:
 
 ``` bash
-mage operator:buildGo
+make build
 ```
 
+Or just ``make``.
+
 ### Testing
 
+Tests are separated to unit-tests (including envtests) and end-to-end tests. To run the unit tests and envtests, use:
```

Comment:
Might be good to put a link to the section in the kubebuilder docs that cover envtest.